### PR TITLE
docs(Input): refactor, add trailing . in description, fix imports

### PR DESCRIPTION
**Pull request by dpkwhan**
- refactor
- add trailing . in description
- fix imports
- fix wrong prop in Button

### CODE REVIEW

File: docs/app/Examples/elements/Input/Variations/InputExampleActionDropdown.js [hunk 1]
```diff
@@ -7,13 +7,13 @@ const options = [
   { text: 'Entire Site', value: 'site' },
 ]
 
-const InputActionDropdown = () => (
+const InputExampleActionDropdown = () => (
   <Input
-    action={<Dropdown basic floating options={options} defaultValue='page' />}
+    action={<Dropdown selection basic floating options={options} defaultValue='page' />}
```
Comment (levithomason) [inline docs/app/Examples/elements/Input/Variations/InputExampleActionDropdown.js]: This should not be a`selection` Dropdown, reference: http://semantic-ui.com/elements/input#action

File: docs/app/Examples/elements/Input/Variations/InputExampleActionLabeledButton.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Input } from 'semantic-ui-react'
+
+const InputExampleActionLabeledButton = () => (
+  <Input
+    action={{ color: 'teal', label: 'right', icon: 'copy', content: 'Copy' }}
```
Comment (levithomason) [inline docs/app/Examples/elements/Input/Variations/InputExampleActionLabeledButton.js]: The prop `label` is for defining the label, the prop `labelPosition` is for placing it on the left/right.  This should be `labelPosition: 'right'`.

File: docs/app/Examples/elements/Input/Variations/InputExampleLeftActionLabeledButton.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Input } from 'semantic-ui-react'
+
+const InputExampleLeftActionLabeledButton = () => (
+  <Input
+    action={{ color: 'teal', label: true, icon: 'cart', content: 'Checkout' }}
```
Comment (levithomason) [inline docs/app/Examples/elements/Input/Variations/InputExampleLeftActionLabeledButton.js]: The previous `labeled` prop became `labelPosition`.  This should be `labelPosition: 'left'` which will place the `icon` as a label on the left.

**Review by levithomason — CHANGES_REQUESTED** [review]

File: docs/app/Examples/elements/Input/Variations/InputExampleActionLabeledButton.js (hunk 2 above)
Comment (dpkwhan) [reply docs/app/Examples/elements/Input/Variations/InputExampleActionLabeledButton.js]: Fixed.

File: docs/app/Examples/elements/Input/Variations/InputExampleLeftActionLabeledButton.js (hunk 3 above)
Comment (dpkwhan) [reply docs/app/Examples/elements/Input/Variations/InputExampleLeftActionLabeledButton.js]: Fixed.

File: docs/app/Examples/elements/Input/Variations/InputExampleRightLabeled.js [hunk 4]
```diff
@@ -7,12 +7,12 @@ const options = [
   { text: '.org', value: '.org' },
 ]
 
-const InputRightLabeled = () => (
+const InputExampleRightLabeled = () => (
   <Input
-    label={<Dropdown defaultValue='.com' options={options} />}
+    label={<Dropdown selection defaultValue='.com' options={options} />}
```
Comment (levithomason) [inline docs/app/Examples/elements/Input/Variations/InputExampleRightLabeled.js]: The `selection` prop should also not be added here:

### with `selection`

image

### without `selection`

image

**Review by levithomason — CHANGES_REQUESTED** [review]
One more update please, then I think we can merge it.

**Comment by dpkwhan** [comment]
@levithomason removed prop selection from Dropdown. Thank you for pointing this.

**Comment by levithomason** [comment]
Thanks much :)

**Pull request merged**